nearcore/src/config.rs

```diff
@@ -206,6 +206,9 @@ pub struct Network {
     pub external_address: String,
     /// Comma separated list of nodes to connect to.
     pub boot_nodes: String,
+    /// Comma separated list of whitelisted nodes. Inbound connections from the nodes on
+    /// the whitelist are accepted even if the limit of the inbound connection has been reached.
+    pub whitelist_nodes: String,
```

Comment:
what is the format?  please add example to the comment above.

Reply:
done